src/components/table.ts

```diff
@@ -51,14 +51,16 @@ export module Component {
       this.classed("table", true);
       rows.forEach((row, rowIndex) => {
         row.forEach((component, colIndex) => {
-          this.addComponent(rowIndex, colIndex, component);
+          if (component != null) {
```

Comment:
This is just an optimization. The case is caught later in the abstractComponentContainer._addComponent(). Not sure what you mean with testing